src/pages/Admin/Opportunity/OpportunityAdminAuthorizationPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { Container } from '@material-ui/core';
+import React, { FC, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import EditMembers from '../../../components/Admin/Community/EditMembers';
+import Loading from '../../../components/core/Loading/Loading';
+import OpportunityMembers from '../../../containers/opportunity/OpportunityMembers';
+import { useOpportunity, useUpdateNavigation } from '../../../hooks';
+import { useOpportunityMembersQuery } from '../../../hooks/generated/graphql';
+import { AuthorizationCredential } from '../../../models/graphql-schema';
+import OpportunityAuthorizationPageProps from './OpportunityAuthorizationPageProps';
+
+export const OpportunityAdminAuthorizationPage: FC<OpportunityAuthorizationPageProps> = ({ paths }) => {
+  const { t } = useTranslation();
+
+  const currentPaths = useMemo(
+    () => [
+      ...paths,
+      {
+        value: '',
+        name: t(`common.enums.authorization-credentials.${AuthorizationCredential.OpportunityAdmin}.name` as const),
+        real: false,
+      },
+    ],
+    [paths]
+  );
+
+  useUpdateNavigation({ currentPaths });
+
+  const { ecoverseId, opportunityId } = useOpportunity();
+
+  const { data, loading } = useOpportunityMembersQuery({ variables: { ecoverseId, opportunityId } });
```

Comment:
you can use the generic `communityMembers` query rather than this one or stick to the same query for different entities, I am not sure which is the better approach.